tests/integration/acp.go

```diff
@@ -215,7 +215,12 @@ func addDocActorRelationshipACP(
 	}
 
 	if action.ExpectedError == "" && !action.ExpectedExistence {
-		waitForUpdateEvents(s, actionNodeID, map[string]struct{}{docID: {}})
+		expect := make([]map[string]struct{}, action.CollectionID+1)
```

Comment:
question: You prefer creating a `CollectionID+1` lenght array that will have just one item instead of passing the collection ID to the `waitForUpdateEvents` function?

Reply:
I started working from the `waitForUpdateEvents` (and late, as a PR fixup).  `waitForUpdateEvents` was multi-collection friendly and so I preserved that without thinking too hard about it, although I think you are right and that all the callers are dealing with a single collection.

Your suggestion makes a lot of sense :)  Although I think I might be able to pass a collection instead of just the index.

- [ ] Simplify `waitForUpdateEvents`